lanl/docker-compose/generate-configs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Ochami config file location
+OCHAMI_CONFIG=/etc/ochami/ochami-config.yaml
+
+#Set DB passwords 
+echo "POSTGRES_PASSWORD=$(openssl rand -base64 32)" > .env
+echo "BSS_POSTGRES_PASSWORD=$(openssl rand -base64 32)" >> .env
+echo "OCHAMI_CONFIG=$OCHAMI_CONFIG" >> .env
+
+#Copy Config template to real config location
+mkdir -p /etc/ochami
```

Comment:
I'm a bit worried about this running (and erring if the user does not have privileges) if the user changes `OCHAMI_CONFIG`. Perhaps you could have:

```bash
OCHAMI_DIR=${OCHAMI_DIR:-/etc/ochami}
OCHAMI_CONFIG=${OCHAMI_CONFIG:-${OCHAMI_DIR}/ochami-config.yaml}
```

and then do

```bash
mkdir -p ${OCHAMI_DIR}
```

instead.

The `${<varname>:-<text>}` is substituted with `<text>` if the environment variable `<varname>` is not set. That way, a user could override those variables.

All this if you would rather not use positional arguments or flags. 😉

Reply:
Great suggestion. added that in

Reply:
Looks good!